### ISSUE TITLE
Calling "define_data_var" with an empty string causes crash

### ISSUE THREAD

**Issue by ZetaTwo** [Vector35/binaryninja-api#3733]
**Version and Platform (required):**
- Binary Ninja Version: 3.2.3913-dev (Build ID f4a307d9)
- OS: Ubuntu Linux
- OS Version: 22.04
- CPU Architecture x64

**Bug Description:**

The following Python call crashes Binary Ninja:
```
bv.define_data_var(0x142ac5658, "")
```

**Steps To Reproduce:**

Run the following in the Python console:
```
bv.define_data_var(0x142ac5658, "")
```

**Expected Behavior:**

Either nothing to happen, the exiting type at the address to be undefined or an error

**Screenshots:**
N/A

**Comment by CouleeApps** [comment]
Likely the same bug as #3727

**Comment by CouleeApps** [comment]
Will be fixed by the same changes that fixed #3727
